sample/isolate.css.scss

```diff
@@ -0,0 +1,98 @@
+
+@import '../grid-system.scss';
+
+.app {
+	@include vui-gs-container;
+}
+
+.app * {
```

Comment:
This is simply a test sample to compare float and isolate output.  Isolate output is helpful to reduce sub-pixel rounding which can be problematic.  In this case however, the float output looks pretty good - I think browsers may have improved a bit in this regard.